app/src/main/AndroidManifest.xml

```diff
@@ -1,20 +1,18 @@
 <?xml version="1.0" encoding="utf-8"?>
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
     package="com.ciphereck.viralinfectiontracker">
-
-<!--    Permissions-->
-    <uses-permission android:name="android.permission.INTERNET" />
-
+    <!-- Permissions -->
     <application
         android:allowBackup="true"
         android:icon="@mipmap/ic_launcher"
         android:label="@string/app_name"
         android:roundIcon="@mipmap/ic_launcher_round"
         android:supportsRtl="true"
         android:theme="@style/AppTheme">
-        <activity android:name=".MainActivity"></activity>
-        <activity android:name=".reports.ReportsActivity"></activity>
-        <activity android:name=".aboutUs.AboutUs"></activity>
+        <activity android:name=".reports.ReportsNavigation"></activity>
+        <activity android:name=".MainActivity" />
+        <activity android:name=".reports.ReportsActivity" />
+        <activity android:name=".aboutUs.AboutUs" />
         <activity
```

Comment:
same module/package activity should be together and must grow like tree instead of randomly